pages/web/hosting/hosting_migrating_to_ovh/guide.it-it.md

```diff
@@ -1,184 +1,258 @@
+> [!primary]
+>
+> Per effettuare questa operazione, è necessario possedere le credenziali di accesso di tutti i tuoi vecchi account email e il nome del server email del tuo precedente provider.
+>
+> Se i tuoi indirizzi email fossero configurati in POP senza conservare copie delle email sul tuo vecchio server di posta, o se disponessi delle email registrate "in locale" sui tuoi dispositivi, potrà essere realizzata solo l'**opzione 2**.
+>
+
+**Opzione 2**: effettua un backup del contenuto dei tuoi indirizzi email con un client di posta (Outlook, Mail di Mac, ecc...), riconfigura il tuo client di posta e importa il backup nel tuo nuovo indirizzo email OVHcloud.
+
+### Step 8: riconfigurare il tuo client di posta <a name="step8"></a>
+
+Una volta migrati i vecchi indirizzi email in OVHcloud, è possibile riconfigurare i client di posta utilizzando tutte le nostre guide disponibili online.
 
-#### Step 4: modifica la configurazione del dominio
+#### Per gli account email "MX Plan": 
 
-Adesso che hai trasferito il tuo sito Web e creato i tuoi indirizzi email presso OVH, è necessario modificare la configurazione del tuo dominio. Per effettuare questa operazione, modifica i server DNS del tuo dominio con quelli di OVH attraverso lo [Spazio Cliente OVH](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.it/&ovhSubsidiary=it){.external}.
+- Consulta la guida Per maggiori informazioni sui parametri di configurazione [Gestisci le email MX Plan](https://docs.ovh.com/it/emails/informazioni-generali-email-condivise/#2-utilizza-il-software-che-preferisci). e i link alle guide personalizzate per i principali client di posta.
 
- Per maggiore assistenza durante questa operazione, consulta la guida relativa a [Informazioni generali sui server DNS](https://docs.ovh.com/it/domains/web_hosting_gestisci_il_tuo_server_dns/){.external}.
+#### Per gli account "Email-Pro":
 
-Questa azione può avere diverse conseguenze:
+- Consulta le nostre guide all'aiuto alla configurazione nelle sezioni `Configurazione su computer` e `Configurazione su smartphone` de [la nostra guida sull'offerta Email-Pro](https://docs.ovh.com/it/emails-pro/).
 
-- **vincolare tecnicamente il tuo dominio alle soluzioni OVH**: il tuo hosting OVH sarà utilizzato per mostrare il tuo sito Web e la ricezione dei nuovi messaggi avverrà sui tuoi indirizzi email creati presso OVH
-- **risolvere un’interruzione del servizio**: se il tuo Registrar ha interrotto la propria configurazione DNS durante il trasferimento del dominio, ciò gli consentirà a quest’ultimo di essere nuovamente raggiungibile.
+#### Per gli account email "Exchange":
+
+- Tutte le nostre guide all'aiuto alla configurazione sono disponibili nelle sezioni `Configurazione Exchange su` e `Configurazione Exchange su smartphone` di [la nostra guida sull'offerta Exchange](https://docs.ovh.com/it/microsoft-collaborative-solutions/).
```

Comment:
Translation of `Configuration Exchange sur ordinateur` is only partial here. Please check https://docs.ovh.com/it/microsoft-collaborative-solutions/ (we should improve sections BTW)

Reply:
Configurazione di Exchange sul computer

Reply:
done